app/models/test.rb

```diff
@@ -5,7 +5,18 @@ class Test < ApplicationRecord
   has_many :users, through: :user_test, dependent: :destroy
   has_many :questions, dependent: :destroy
 
-  def self.by_category_title(category_title)
-    joins(:category).where(categories: { title: category_title }).order(title: :desc).pluck(:title)
+  scope :easy, -> { where(level: 0..1) }
+  scope :medium, -> { where(level: 2..4) }
+  scope :hard, -> { where(level: 5..Float::INFINITY) }
+  scope :by_category_title, lambda { |title|
+    joins(:category).where(categories: { title: title }).order(title: :desc)
+  }
+
+  validates :title, presence: true
+  validates :title, uniqueness: { scope: :level }
+  validates :level, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+
+  def self.by_category_title(title)
+    joins(:category).where(categories: { title: title }).pluck(:title)
```

Comment:
```suggestion
    by_category_title(title).pluck(:title)
```
у тебя же скоуп есть